devp2p-eth/src/test/kotlin/org/apache/tuweni/devp2p/eth/EthHandlerTest.kt

```diff
@@ -128,22 +128,22 @@ class EthHandlerTest {
         controller = EthController(repository, MemoryTransactionPool(), requestsManager)
       )
 
-      for (i in 1..10) {
-        val newBlock = createChildBlock(header)
-        repository.storeBlock(newBlock)
-        var txIndex = 0
-        for (tx in newBlock.body.transactions) {
-          repository.storeTransactionReceipt(
-            TransactionReceipt(
-              Bytes32.random(),
-              32L, LogsBloomFilter(), emptyList()
-            ),
-            txIndex, tx.hash, newBlock.header.hash
-          )
-          txIndex++
-        }
-        header = newBlock.header
-      }
+
+      val newBlock = createChildBlock(header)
+      repository.storeBlock(newBlock)
+      var txIndex = 0
+
+      repository.storeTransactionReceipt(
+        TransactionReceipt(
+          Bytes32.random(),
+          32L, LogsBloomFilter(), emptyList()
+        ),
+        txIndex, newBlock.body.transactions.hash, newBlock.header.hash
```

Comment:
this is not going to compile, the list of transactions doesn't have a hash, each transaction has its own hash.
Is there a reason for modifying this test? Seems unrelated to your changes.